### ISSUE TITLE
Why is the build method building AND showing the showcase view?

### ISSUE THREAD

**Issue by bhargavms** [amlcurran/ShowcaseView#360]
Why is the build method building AND showing the showcase view? instead of just return ShowCaseView?

**Comment by amlcurran** [comment]
:+1: an excellent point! In some branch I have somewhere, I have in fact deprecated `build` for a more explicit `buildAndShow` method. Can't find it at the moment but will hopefully have this pushed soon. Completely agree with you though!

**Comment by bhargavms** [comment]
but why deprecate build? why do you need to buildAndShow() and the same time?

**Comment by amlcurran** [comment]
Because if someone is currently using `build` knowing it will show, and then suddenly I change it so that it doesn't show when I update the version, it breaks the API without them really being aware.

**Comment by bhargavms** [comment]
if thats the case then I suggest leaving build in there too, because if I wanted to show multiple targets one after the other I would need the build method

**Comment by amlcurran** [comment]
yeah, I am currently working out the best way to do it, but I'm aware :)
